lib/src/components/NxPolicyViolationIndicator/types.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import * as PropTypes from 'prop-types';
+import { HTMLAttributes, WeakValidationMap } from 'react';
+
+import { ThreatLevelNumber, ThreatLevelCategory, allThreatLevelCategories, allThreatLevelNumbers }
+  from '../../util/threatLevels';
+
+export interface Props extends HTMLAttributes<HTMLDivElement> {
+  threatLevelCategory?: ThreatLevelCategory;
+  policyThreatLevel?: ThreatLevelNumber;
+}
```

Comment:
Given that we really only want strings for the children, we can signify that by adding `children?: string | null;` here. And then `children: PropTypes.string` in the propTypes below.

Reply:
> Given that we really only want strings for the children, we can signify that by adding `children?: string | null;` here. And then `children: PropTypes.string` in the propTypes below.

https://github.com/sonatype/sonatype-react-shared-components/pull/229/commits/bec43678f47e6988702a7b94e7b826ff7643b26d